common/presenters/person-to-summary-list-component.test.js

```diff
@@ -1,3 +1,4 @@
+const { API } = require('../../config')
```

Comment:
Should this be mocked so the two branches can be tested in isolation?

Reply:
I don't think it's worth the effort. It should only run as one or the other and the forking stuff should be yanked out the moment we switch over and are happy everything's ok. Unless you have a way of running the e2es with the VERSION mocked too ;)